### ISSUE TITLE
Duplicate Subject header causes failure sending to AWS

### ISSUE THREAD

**Issue by marvel16** [lukencode/FluentEmail#228]
Mailkit sender bug ?

![image](https://user-images.githubusercontent.com/5057793/99772148-e9fcda00-2b12-11eb-83cd-6317fe2a1764.png)

**Comment by she2** [comment]
I think this is the main reason emails to yahoo addresses refused to deliver. It returned this error

*554 Message not allowed - Headers are not RFC compliant[291]*

**Comment by aphagel** [comment]
Confirmed. I encountered the same issue today. Email was rejected because of: 550 Messages should have one or no Subject headers, not 2.

The (my) issue was solved after making a local copy of the FluentEmailKit and removing the second line marked in the image of marvel16.

Note: when create a new instance of the 'MimeMessage' class the collection of headers immediately contains a header for the content (empty string value). When you assign a value to the message.Subject property then the MimeMessage object automatically overwrites the (existing!) header. The line 'message.Headers.Add(HeaderId.Subject' adds a second header and for that reason (some) mailservers reject the e-mail.

I propose to remove to line since it is not necessary. If you want to ensure that the headers contain a subject header: verify if a subject header exists and if exists then overwrite the value or add the header (with value) when it does not exist.
